python/ray/tests/test_traceback.py

```diff
@@ -352,11 +327,8 @@ def f():
     try:
         ray.get(f.remote())
     except Exception as ex:
-        python310_extra_exc_msg = "test_unpickleable_stacktrace.<locals>.NoPickleError."
-        cleaned = scrub_traceback(str(ex)).replace(
-            f"TypeError: {python310_extra_exc_msg}", "TypeError: "
-        )
-        assert clean_noqa(expected_output) == cleaned
+        assert isinstance(ex, UnserializableException)
+        assert clean_noqa(expected_output) == scrub_traceback(str(ex))
```

Comment:
you can use `with pytest.raises` for this

the way the test is currently written is incorrect -- if the `ray.get` doesn't raise an exception, it will pass